src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/TracerProviderBuilderExtensions.cs

```diff
@@ -29,11 +31,19 @@ public static class TracerProviderBuilderExtensions
         /// Add AWS Lambda configurations.
         /// </summary>
         /// <param name="builder"><see cref="TracerProviderBuilder"/> being configured.</param>
+        /// <param name="configure">AWS lambda instrumentation options.</param>
         /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
-        public static TracerProviderBuilder AddAWSLambdaConfigurations(this TracerProviderBuilder builder)
+        public static TracerProviderBuilder AddAWSLambdaConfigurations(
```

Comment:
This method name is weird. But since it was there before, fine with me.

Reply:
Yes, but unfortunately this is a part of public API.